.github/workflows/black.yml

```diff
@@ -0,0 +1,12 @@
+# This workflow will lint code changes using black when code is pushed to the repository and when making pull requests
+# For more information see: https://black.readthedocs.io/en/stable/integrations/github_actions.html
+name: Lint
+
+on: [push, pull_request]
```

Comment:
Should it be on push and pull requests only or is it preferred to be on different options?